### PR TITLE
feat: Add pagopa.it helm chart

### PR DESCRIPTION
**Pull request by matteocellucci**

### CODE REVIEW

File: pagopa_it/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: pagopa-it-certificate-check
+description: Check the pagopa.it TLS/SSL certificate expiration date
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+dependencies:
+- name: microservice-chart
+  version: 1.17.0
```
Comment (diegolagospagopa) [inline pagopa_it/Chart.yaml:9]: Can you use the version `1.20`?

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: pagopa_it/Chart.yaml (hunk 1 above)
Comment (matteocellucci) [reply pagopa_it/Chart.yaml:9]: This is temporary. I'm going to update the template and I'll have to use a version above `1.20`

Comment (matteocellucci) [reply pagopa_it/Chart.yaml:9]: We need to handle the image pull with secrets

Comment (diegolagospagopa) [reply pagopa_it/Chart.yaml:9]: OK

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**